### ISSUE TITLE
Pick listing format for officers

### ISSUE THREAD

**Issue by mjcollin** [dsiufl/www.dsiufl.org-2016#1]
Some options for listing officers:

1. List everyone once. Have current officers at the top and then prior officers below in order by the time they served last. Use a "semester" variable like "2016 Spring" and a "sort" variable like "3" for ordering both lists.

2. List groupings of officers by semester. This addresses the issue of people who have multiple positions over time without having to make lists of positions in people's profile. This does require editing the about.html template every semester to add another grouping or some more creative Jekyll work. Have a "semester" and "sort" still.

3. Something else?

**Comment by xiaoyaya** [comment]
2.1 Or we can list groupings of officers by semester this way. Just use the current tag to differentiate current semester (for "2016 Spring") with previous semesters (for "2015 Fall" and "2015 Spring"). Then under "Previous Semesters" section, we sort by year, semester and positions (variable like "3"). So about.html doesn't need to be overwrite. We just need to change the tag from current semester to previous semester then. 
Basically, the format would like: 
current: true  // use this to determine which section it goes
Semester: 2016 Spring

**Comment by mjcollin** [comment]
If we do it that way, it will be similar to 1. The main difference between 1. and 2. in my mind is whether we repeat officers who have been elected multiple times. To repeat the display of an officer, there needs to be a loop for each section:
```
for officer in 2016_Spring:
  show officer
for officer in 2015_Fall:
  show officer
...
```

verses
```
for officer in current:
  show officer
for officer in !current:
  show officer
```

Where officer looks like:
```
current: true
2016_Spring: true
2015_Fall: false
```

**Comment by xiaoyaya** [comment]
Now I get what you mean :)
Maybe option 1? Then they don't have to write on about.html.
And they have the option to transfer to option 2 if they want to play with some coding?
